tests/test_manager.py

```diff
@@ -295,7 +296,7 @@ def mkdtemp():
                 "method": "PUT",
                 "body": {"permission": "push"},
             },
-            # set user on repo
+            # add user on repo
```

Comment:
Am I wrong or this test is not complete? 
I don't see where you add/remove ppl in the conf file.